examples/contracts_for_symbol/index.js

```diff
@@ -0,0 +1,46 @@
+import DerivAPIBasic from "https://cdn.skypack.dev/@deriv/deriv-api/dist/DerivAPIBasic";
```

Comment:
better to add a empty line after imports

Reply:
fixed 👍